src/main/webapp/scripts/tests/controller-auth-test.js

```diff
@@ -0,0 +1,93 @@
+import * as view from '/scripts/view.js';
+import * as controller from '/scripts/controller-auth.js';
+
+// Test (controller-auth.js and) view.js
+QUnit.module('Authorization controller', {
+  beforeEach: () => {
+    gapi = new CustomGapi();
+  },
+  afterEach: () => {
+    view.updateSigninStatus(false, null, null, null);
+    document.getElementById('result-text').innerText = '';
```

Comment:
Is this needed? If it is, maybe put in a beforeEach instead. It makes it clear that the page is reset before starting a test.

Reply:
Done